mappings/net/minecraft/client/gui/EditBox.mapping

```diff
@@ -0,0 +1,170 @@
+CLASS net/minecraft/class_7530 net/minecraft/client/gui/EditBox
+	COMMENT A multiline edit box with support for scrolling and basic keyboard shortcuts.
+	COMMENT This should not be used directly; {@link net.minecraft.client.gui.widget.EditBoxWidget}
+	COMMENT should be used instead.
+	FIELD field_39511 UNLIMITED_LENGTH I
+		COMMENT A constant denoting that the edit box accepts unlimited amount of text. Edit box
+		COMMENT widgets with such edit boxes do not show the current text length indicator.
+	FIELD field_39512 CURSOR_WIDTH I
+	FIELD field_39513 textRenderer Lnet/minecraft/class_327;
+	FIELD field_39514 lines Ljava/util/List;
+	FIELD field_39515 currentValue Ljava/lang/String;
```

Comment:
IMO it'd be better to call this current "text" than the "value" of an edit box, which is much more vague.

Reply:
Okay, changing. Seems like Swing uses text too.

Reply:
Yeah, that's where I got it from :smile: